frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -440,7 +442,19 @@ export class NativeQueryEditor extends Component<
       showLineNumbers: true,
     });
 
-    this._lastAutoComplete = { timestamp: 0, prefix: null, results: [] };
+    let lastAutoComplete: LastAutoComplete = {
+      timestamp: 0,
+      prefix: "",
+      results: [],
+      first: true,
+    };
+
+    const resultsForAce = (results: [string, string][]) =>
+      results.map(([name, meta]) => ({
+        name: name,
+        value: name,
+        meta: meta,
+      })) ?? [];
```

Comment:
I doubt `?? []` will ever be called
```suggestion
      }));
```